build.gradle

```diff
@@ -37,9 +37,10 @@ dependencies {
     compileOnly "com.google.code.findbugs:jsr305:3.0.2"
 
     // Databreaker/optimization mods
-    modRuntime "com.github.jellysquid3:sodium:1.16.x~stable-SNAPSHOT"
-    modRuntime "com.github.jellysquid3:lithium:1.16.x~dev-SNAPSHOT"
-    modRuntime ("com.github.SuperCoder7979:databreaker:${databreaker_version}") {
+    modRuntime "com.github.jellysquid3:sodium:${project.sodium_version}"
```

Comment:
I have changed this to be more correct in this way locally, if you could remove the build.gradle/gradle.properties changes that would be great.

Reply:
Okay, I will remove them.